src/15utility.js

```diff
@@ -492,9 +504,18 @@ var loadBinaryFile = utils.loadBinaryFile = function(path, asy, success, error)
                 success(arr.join(""));
             }
         }
-		//*/
+	} else if(utils.isReactNative) {
+        /* If ReactNative */

```

Comment:
As this is also stated in the `If` just above, I recommend ` /* If ReactNative */`gets removed